### DISCUSSION TITLE
Slack channel joining problem

### DISCUSSION THREAD

**Discussion by JigarJoshi04** [thomaspoignant/go-feature-flag#1290] [Q&A] [answered]
How to join the slack community as it gives error saying <gamil account >doesn’t have an account on this workspace.

**Comment by thomaspoignant** [comment, marked as answer]
@JigarJoshi04 Sorry about the delay in my response.
To join the Slack channel you can use this URL https://invite.slack.golangbridge.org/ to get an invite.
After you can join `#go-feature-flag`.

**Reply by thomaspoignant** [reply]
FYI we've updated the Slack page on the website to have that information and you can find it here : https://gofeatureflag.org/slack
